packages/top-app-bar/constants.js

```diff
@@ -0,0 +1,4 @@
+export const cssClasses = {
```

Comment:
I dont think this needs to be a separate file....does anything besides packages/top-app-bar/index.js use this?

Reply:
no i will remove. I was using it in the tests